interfaces/Config/templates/config_sorting.tmpl

```diff
@@ -156,6 +171,43 @@
                                     <td>$T('text')</td>
                                 </tr>
                             </tbody>
+                            <tbody>
+                                <tr>
+                                    <th class="align-right"><b>GuessIt</b></th>
+                                    <th>$T('sort-pattern')</th>
+                                    <th>$T('sort-result')</th>
+                                </tr>
+                            </tbody>
+                            <tbody>
+                                <tr>
+                                    <td class="align-right"><b>$T('sort-guessitMeaning'):</b></td>
+                                    <td>%GI&lt;$T('sort-guessitProperty')&gt;</td>
+                                    <td>$T('guessit-sp-property')</td>
+                                </tr>
+                                <tr>
+                                    <td class="align-right"><b></b></td>
+                                    <td>%G.I&lt;$T('sort-guessitProperty')&gt;</td>
+                                    <td>$T('guessit-dot-property')</td>
+                                </tr>
+                                <tr>
+                                    <td class="align-right"><b></b></td>
+                                    <td>%G_I&lt;$T('sort-guessitProperty')&gt;</td>
+                                    <td>$T('guessit-us-property')</td>
+                                </tr>
+                                <tr>
+                                    <td class="align-right"><b>$T('Example')</b></td>
+                                    <td>%GI&lt;audio_codec&gt;</td>
+                                    <td>DTS</td>
+                                </tr>
+                                <tr>
+                                    <td class="align-right" style="vertical-align:top;"><b>$T('Valid properties')</b></td>
+                                    <td colspan=2 style="word-spacing: 0.7em; max-width:275px; white-space: wrap;">
+                                        <!--#for $prop in $guessitProperties#-->
+                                        $prop
```

Comment:
I initially did just that, but that was quite long (vertically) while leaving lots of empty space. Idealy, this would have multiple columns, but my cheetah skills fell short.

Reply:
Switched back to a linebreak for now.